src/components/app/app.js

```diff
@@ -32,7 +32,6 @@ const App = () => {
 
   useEffect(() => {
     dispatch(getCategories());
-    dispatch(getContacts());
```

Comment:
Why have you removed this line?

Reply:
I moved this line to the app file so that the contact information is loaded as soon as the page starts. In another case the request is made only after going to the contact page. This will also come in handy when the footer will fixed and this data is taken, as there are now stubs.